tests/test-cases/Group1-Docker-Commands/1-04-Docker-Create.robot

```diff
@@ -120,8 +120,7 @@ Create a container with no command specified
 Create a container with custom CPU count
     ${rc}  ${id}=  Run And Return Rc And Output  docker %{VCH-PARAMS} create -it --cpuset-cpus 3 busybox
     Should Be Equal As Integers  ${rc}  0
-    ${rc}  ${id}=  Run And Return Rc And Output  govc datastore.ls |grep ${id}
-    Should Be Equal As Integers  ${rc}  0
+    ${id}=  Get VM display name  ${id}
```

Comment:
@emlin `Get VM display name  ${id}` returns `containerName-containerShortID`. This test case should be fine. I have tested it.